packages/manager/modules/web-universe-components/src/order-cart/order-cart.service.js

```diff
@@ -98,6 +180,12 @@ export default class OrderCartService {
       }).$promise;
   }
 
+  /**
+   * Get the informations for checkout a order
```

Comment:
```suggestion
   * Get the information for checkout a order
```
"Fun" fact : `information` is invariable :)